src/apps/openfluid-devstudio/MainWindow.cpp

```diff
@@ -744,14 +603,50 @@ void MainWindow::onExportWareSourcesAsked()
 // =====================================================================
 
 
+void MainWindow::applyBuildOptionsToWidgetCollection()
```

Comment:
In a correct workflow, we should not update each item of the Widget collection with the new build options, but each widget should use common build options stored in only one place. Can be held by the WorkspaceManager?
This modification is maybe too deep to be implemented in this PR?